src/test/java/TiTi/task/DeadlineTest.java

```diff
@@ -0,0 +1,22 @@
+package TiTi.task;
+
+import TiTi.task.Deadline;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DeadlineTest {
+
+    @Test
+    public void basicTest(){
+        Deadline testDeadline = new Deadline("homework", "Mon 5pm");
+        assertEquals(testDeadline.toString(), "[D][ ] homework (by: Mon 5pm)");
+    }
+
+    @Test
+    public void DateTest(){
```

Comment:
According to coding standards, names representing methods must be written in camelCase. So perhaps `dateTest`. Anyway, this part is probably overlooked as well.